content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -0,0 +1,207 @@
+---
+reviewers:
+- liggitt
+- mikedanese
+- munnerz
+title: Certificate Signing Requests
+content_template: templates/concept
+weight: 20
+---
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+The Certificates API enables automation of
+[x509](https://tools.ietf.org/html/rfc5280) credential provisioning by providing
+a programmatic interface for clients of the Kubernetes API to request and obtain
+x509 certificates from a Certificate Authority (CA).
+
+{{% /capture %}}
+
+{{% capture body %}}
+## Certificate Signing Requests Overview
```

Comment:
It's a sort of antipattern to close the overview section and then have a heading that says it's an overview.

To address that, I recommend summarising the next 3 paragraphs and moving them into the overview, and then revise the heading to something like “Request signing process”.

(If anyone wants to make a diagram, I think that would really help - that'd be a separate PR BTW).